src/features/left-sidebar/LeftSidebar.js

```diff
@@ -185,7 +185,7 @@ class LeftSidebar extends React.Component<Props, State> {
 
     getIcon(
         iconElement?: ?React.Element<any>,
-        IconComponent?: ?React.ComponentType<any>,
+        IconComponent?: ?React.ComponentType<any>, // eslint-disable-line
```

Comment:
not sure why this was giving a problem with new eslint, but this was the only occurrence